scripts/lint-copy-mod-doc.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+
+import sys
+
+fns = sys.argv[1:]
+
+ERR_COP = 0
+ERR_IMP = 1
+ERR_MOD = 2
+
+exceptions = []
+
+with open("scripts/copy-mod-doc-exceptions-short.txt") as f:
+    lines = f.readlines()
+    for line in lines:
+        (fn, c, errno) = line.split()
+        if errno == "ERR_COP":
+            exceptions += [(ERR_COP, fn)]
+        if errno == "ERR_IMP":
+            exceptions += [(ERR_IMP, fn)]
+        if errno == "ERR_MOD":
+            exceptions += [(ERR_MOD, fn)]
+
+new_exceptions = False
+
+def import_only_check(lines, fn):
+    import_only_file = True
+    errors = []
+    line_nr = 0
+    for line in lines:
+        line_nr += 1
+        if line == "\n":
+            continue
+        imports = line.split()
+        if imports[0] != "import":
+            import_only_file = False
+            break
+        if len(imports) > 2:
+            if imports[2] == "--":
+                continue
+            else:
+                errors += [(ERR_IMP, line_nr, fn)]
+    return (import_only_file, errors)
+
+def regular_check(lines, fn):
+    line_nr = 0
+    errors = []
+    copy_started = False
+    copy_done = False
+    copy_start_line_nr = 0
+    for line in lines:
+        line_nr += 1
+        if not copy_started and line == "\n":
+            continue
+        if not copy_started and line == "/-\n":
+            copy_started = True
+            copy_start_line_nr = line_nr
+            continue
+        if not copy_started:
+            errors += [(ERR_COP, line_nr, fn)]
+        if copy_started and not copy_done:
+            if line_nr == copy_start_line_nr + 1 and not "Copyright (c)" in line:
+                errors += [(ERR_COP, line_nr, fn)]
+            if line_nr == copy_start_line_nr + 2 and not "Apache" in line:
+                errors += [(ERR_COP, line_nr, fn)]
+            if line_nr == copy_start_line_nr + 3 and not ("Authors:" in line or "Author:" in line):
```

Comment:
```suggestion
            if line_nr == copy_start_line_nr + 3 and not "Author" in line:
```
There is also a bunch of `Author(s):`